testnets/artelatestnet/assetlist.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "artelatestnet",
+  "assets": [
+    {
+      "description": "The native EVM, governance and staking token of the Artela Network",
+      "denom_units": [
+        {
+          "denom": "uart",
+          "exponent": 0
+        },
+        {
+          "denom": "art",
+          "exponent": 18
+        }
+      ],
+      "base": "uart",
+      "name": "Artela Testnet",
+      "display": "art",
+      "symbol": "ART",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/artelatestnet/images/artela.png"
```

Comment:
Do you have a square aspect ratio token logo?

Reply:
The current image should be used for a chain logo defined in chain.json (but not defined, currently)

Reply:
thanks Jeremy, I will add new logo

Reply:
hi Jeremy, new logo has updated

Reply:
@JeremyParish69 hi Jeremy, new logo has updated, thanks, please review